cmd/promtool/main.go

```diff
@@ -501,10 +501,15 @@ func QueryLabels(url *url.URL, name string, p printer) int {
 	// Run query against client.
 	api := v1.NewAPI(c)
 	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
-	// TODO: Show warnings (second return value) somehow? See https://github.com/prometheus/prometheus/issues/5885
-	val, _, err := api.LabelValues(ctx, name)
+	val, warn, err := api.LabelValues(ctx, name)
 	cancel()
 
+	if warn != nil {
```

Comment:
How about omitting the `if` clause completely because the loop over a `nil` or empty slice will just do nothing anyway.

Reply:
Yeah, makes sense.